assets/stylesheets/overall.scss

```diff
@@ -35,6 +35,7 @@ body {
     width: 100%;
     // set the fixed height of the footer here
     height: 30px;
+    align-content: end;
```

Comment:
my firefox developer tools say about this
> align-content has no effect on this element since it’s neither a flex container nor a grid container.
> Try adding display:grid or display:flex.

Reply:
yes i know, i forgot to add this variable

Reply:
And does this actually something for you? For me it makes no difference in display